library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -287,4 +296,37 @@ public void onMapClick(LatLng latLng) {
   public GoogleMap getGoogleMap() {
     return googleMap;
   }
+
+  @Override
+  public void setGeoJsonLayer(AirMapGeoJsonLayer airMapGeoJsonLayer) {
+    // clear any existing layers
+    clearGeoJsonLayer();
+
+    try {
+      layerOnMap = new GeoJsonLayer(getMap(), new JSONObject(airMapGeoJsonLayer.geoJson));
+    } catch (JSONException e) {
+      Log.e(TAG, "Failed to add GeoJson layer", e);
```

Comment:
The web one silently fails so I did the same here for consistency of API, but lemme add it atleast for native.